README.md

```diff
@@ -16,6 +15,8 @@ It features:
 * Testing your project.
 * Benchmarking your project.
 
+`stack` replaces `stackage-cli`. For more details on the motivation for the creation of `stack`, a comparison of `stack` to other tools, and the roadmap for the future of `stack', [see this document]().
```

Comment:
reference to stackage-cli looks like unneeded trivia to me that most people don't need to know at all.